jaxrs/src/test/java/com/cerner/beadledom/jaxrs/provider/FakeDao.java

```diff
@@ -0,0 +1,8 @@
+package com.cerner.beadledom.jaxrs.provider;
+
+public class FakeDao {
```

Comment:
nit: Can we call this `FakeRepository`?

Reply:
https://github.com/cerner/beadledom/pull/45/commits/b19a43a19a67ff8b1aa1723f780453e7bbeb55ac#diff-eb3b518f028258b8be83878f3ec5ee4eR3